<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="1" hostname="DESKTOP-35HBDVU" failures="2" tests="2" name="Default test(failed)" time="0.05" errors="0" timestamp="2024-02-04T12:18:22 IST">
  <testcase classname="com.app.sampletest.Failed_Test1" name="test1" time="0.009">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at com.app.sampletest.Failed_Test1.test1(Failed_Test1.java:10)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1541)
... Removed 31 stack frames]]>
    </failure>
  </testcase> <!-- test1 -->
  <testcase classname="com.app.sampletest.Failed_Test1" name="test3" time="0.002">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at com.app.sampletest.Failed_Test1.test3(Failed_Test1.java:16)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1541)
... Removed 31 stack frames]]>
    </failure>
  </testcase> <!-- test3 -->
  <testcase classname="com.app.sampletest.Failed_Test1" name="test2" time="0.0">
    <ignored/>
  </testcase> <!-- test2 -->
</testsuite> <!-- Default test(failed) -->
